config/routes.rb

```diff
@@ -0,0 +1,61 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'trips#index'
+
+  resources :trips do
+    resources :passengers, include: [:index, :show]
```

Comment:
Why do we nest passenger routes in trips?

Nesting passenger routes in trips gives us the routes like
`trip_passengers GET    /trips/:trip_id/passengers` ... What does that get us?

Also, is the syntax isn't `include:`, it's `only:`